HM8_task2.py

```diff
@@ -0,0 +1,4 @@
+def square_evens_comp(numbers):
+  return [num*num for num in range(numbers) if num % 2 == 0]
+
+print(square_evens_comp(1000000000))
```

Comment:
The solution should work and not freeze your computer.

When I run this code I do not see any result for more than 1 minute. 
After that I got this in the terminal:
![image](https://user-images.githubusercontent.com/25909810/221840081-074deaa4-69ce-454c-9b5a-357a54cc5bad.png)


Please use an approach where you can print one number after another without freezing python